packages/react/src/components/linkbox/Linkbox.module.scss

```diff
@@ -0,0 +1,103 @@
+.linkbox {
+  cursor: pointer;
+
+  &:focus {
+    outline: var(--color-coat-of-arms) 3px solid;
+  }
+}
+
+.withBackground {
+  background-color: white;
```

Comment:
Should we use `var(--color-white)` here?

Reply:
- [ ] Apply `var(--color-white)` here